i18n/en-GB/cosmic_term.ftl

```diff
@@ -0,0 +1,99 @@
+cosmic-terminal = COSMIC Terminal
+new-terminal = New terminal
+
+# Context Pages
+
+## About
+git-description = Git commit {$hash} on {$date}
+
+## Color schemes
+color-schemes = Colour schemes
+rename = Rename
+export = Export
+delete = Delete
+import = Import
+import-errors = Import errors
+
+## Profiles
+profiles = Profiles
+name = Name
+command-line = Command line
+tab-title = Tab title
+tab-title-description = Override the default tab title
+add-profile = Add profile
+new-profile = New profile
+make-default = Make default
+working-directory = Working directory
+hold = Hold
+remain-open = Remain open after child process exits.
+
+## Settings
+settings = Settings
+
+### Appearance
+appearance = Appearance
+theme = Theme
+match-desktop = Match desktop
+dark = Dark
+light = Light
+syntax-dark = Color scheme dark
```

Comment:
Do you want these to be colour?

Reply:
Yes, it would have helped if I had actually committed the changes I was wanting to make! Fixed now.